CHANGELOG.md

```diff
@@ -2,21 +2,66 @@
 
 ## 3.0.0-alpha.5 - Unreleased
 
+* Conditional fields are now supported. The old 2.x `showFields` feature has been replaced with `if: { ... }`. The new feature works like this:
```

Comment:
Let's move away from detailed documentation examples in the changelog. We should add it to the a3-docs instead.